### ISSUE TITLE
Anti-virus software complains that IncrementalCompiler.exe contains IHeur.EKI.

### ISSUE THREAD

**Issue by antlaw** [SaladLab/Unity3D.IncrementalCompiler#13]
When I unzip the release files for Unity 5, my anti-virus software complains that IncrementalCompiler.exe contains IHeur.EKI and requries me block it. What's that about?

**Comment by veblush** [comment]
Yes it's false positive from unfriendly anti-virus tools :wink:. I've tried something to keep away from them but it seems failed. If you can unblock it, do it first and wait for next release.
